MIP/mip-40/README.md

```diff
@@ -0,0 +1,83 @@
+# MIP-40: *AB-FFS Decoupled*
+- **Description**: Introduces architectural guide for ***AB-FFS Decoupled***. 
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Reviewer**: [Andreas Penzkofer](mailto:andreas.penzkofer@movementlabs.xyz)
+- **Desiderata**: [MD-38](https://github.com/movementlabsxyz/MIP/pull/38)
+
+## Abstract
+
+This MIP introduces the architectural guide for implementing ***AB-FFS Decoupled***, a means of using the Atomic Bridge and Fast-finality Settlement with a fixed supply of token. This decoupled approach relies on burning gas on the L2 into an L2 pool and issuing rewards in an LP token on the L1. Details of the L2 pool recirculation and the L1 LP token issuance are largely left to follow-on MIPs, however we do specify general constraints and guidelines. 
+
+We also propose the usage of intermediary tokens for bridging to fully decouple the L1 and L2 systems. This allows for the L2 pool to be recirculated without the need for a direct bridge to the L1 and exposure to fallibility of the bridge on either side. 
+
+Without the usage of intermediary tokens, we refer to this system as ***AB-FFS Partially Decoupled***, for which we do not intend a separate MIP. 
+
+## Motivation
+
+The *AB-FFS Decoupled* system is intended to address [MD-38](https://github.com/movementlabsxyz/MIP/pull/38). That is, *AB-FFS Decoupled* provides a means for using the Atomic Bridge and Fast-finality Settlement with a fixed supply of token. 
+
+Fundamentally, *AB-FFS Decoupled* contends with the fallibility of the Atomic Bridge by (a) removing any need for its involvement in returning funds to the L1 and (b) using intermediary tokens to avoid any locking, minting, burning, or other operations that would potentially modify the total supply of either the L1 staking token or the L2 staking token.
```

Comment:
So how many different tokens do we have now? 
What are the staking tokens? Validators stake L1MOVE and get L1MOVE rewards.
There should only be two tokens L1MOVE and L2MOVE. 
Safeguards can be added to the MOVE ERC-20 and MOVE FA on L2.

Reply:
Under this proposal, validators stake the L1 staking token (L1MOVE) and get an LP token (not named).

How are you going to ensure the fixed supply we are now required to maintain? If the bridge fails, the supply can increase or decrease.

Reply:
> How are you going to ensure the fixed supply we are now required to maintain? If the bridge fails, the supply can increase or decrease.

this solution does not solve this problem
- on both L1 and L2 the max circulating supply should be capped at MOVE_max ( this seems a natural restriction)
- consequently due to the 1-1 swap component the bridge token should be limited to MOVE_max on either side.
- if the bridge fails at most MOVE_max equivalent of bridge token can be created on either side.
- consequently on L1 MOVE_max L1MOVE can be brought into circulation and MOVE_max L2MOVE can be brought into circulation. doubling the supply. 

This is the same for a direct bridge L1MOVE <> L2MOVE. 

Hence this protocol does not seem to solve the problem it attempts to solve.

Reply:
> How are you going to ensure the fixed supply we are now required to maintain? If the bridge fails, the supply can increase or decrease.

this solution does not solve this problem

- on both L1 and L2 the max supply of L1MOVE and L2MOVE should be capped at MOVE_max ( this seems a natural restriction)
- if the bridge gets compromised an adversary can empty the exchange on both L1 and L2 and bring into circulation 2x MOVE_max 

This is the same for a direct bridge L1MOVE <> L2MOVE, if we can implement a mint restriction that checks the total supply on L2 (similar for L1)

Hence this protocol does not seem to solve the problem it attempts to solve.